src/sql/src/catalog.rs

```diff
@@ -169,6 +169,14 @@ pub trait SessionCatalog: fmt::Debug + ExprHumanizer + Send + Sync {
         cluster_name: Option<&'b str>,
     ) -> Result<&dyn CatalogCluster<'a>, CatalogError>;
 
+    /// Resolves the named cluster replica.
+    ///
+    /// If the provided name is `None`, resolves the currently active cluster.
```

Comment:
I don't see an `Option` anywhere here.

Reply:
Oops, fixed that. There's no option, I just copied and pasted the comment.